### ISSUE TITLE
Support for firebase version 5.3.0?

### ISSUE THREAD

**Issue by 1natsu172** [firebase/firebaseui-web-react#39]
## module versions

* "react-firebaseui": "^3.0.4"
* "firebase": "5.3.0"

---

Hi, I upgraded the project dependencies Firebase from version 5.0.4 to 5.3.0 and I get an error.

**Error Message on React**

`Firebase: Firebase service named 'auth' already registered (app/duplicate-service).`

I think that peerDependencies of  "react-firebaseui" don't support v5.3.0 is the error cause.

```json
 "peerDependencies": {
    "react": ">=15 <17",
    "firebase": ">=5 <=5.0.1"
  },
```

BTW, I'm using TypeScript on the project. So import syntax is as fellows.

```typescript
import firebase from 'firebase/app'
import 'firebase/auth'
import 'firebase/database'
```

**Comment by nicolasgarnier** [comment]
Just released v3.0.6 to remove the restriction.

**Comment by 1natsu172** [comment]
@nicolasgarnier 

Thanks for your support! 🙏 
Currently running in v5.3.1 Firebase without problems in my environment.

FYI, There was a bug ["auth recalled when signing out immediately after logging in"](https://github.com/firebase/firebaseui-web/issues/440).
This problem has been resolved, as this fix has been merged into v5.3.0 Firebase-js-sdk.
I was bothered by this problem very much.
